sql/report_queries/erm/costs/erm_inventory_cost.sql

```diff
@@ -0,0 +1,179 @@
+             inv.approval_date AS "invoice_approval_date"
+         FROM
+             invoice_invoices AS inv
+     )
+SELECT
+    invl.po_line_id,
+    invl.id AS "invl_id",
+    invl.invoice_line_status AS "invl_status",
+    pol.payment_status AS "po_line_payment_status",
+    pol.is_package AS "po_line_is_package",
+    inv.invoice_approval_date AS "invoice_approval_date",
+    pol.order_format AS "po_line_order_format",
+    pol_phys_type.pol_mat_type_name AS "po_line_phys_mat_type",
+    pol_er_type.pol_er_mat_type_name AS "po_line_er_mat_type",
+    inst.instance_mode_of_issuance_name,
+    invl_adjustments.adjustment_description AS "invl_adjustment_description",
+    invl_adjustments.adjustment_prorate AS "invl_adjustment_prorate",
+    invl_adjustments.adjustment_relationtototal AS "invl_adjustment_relationtototal",
+    invl_adjustments.adjustment_value AS "invl_adjustment_value",
+    cast(invl.sub_total AS money) AS "invl_sub_total",
+    cast(invl.total AS money) AS "invl_total", -- this are costs by invoice_line
+    inv_adj.inv_adj_prorate,
+    inv_adj.inv_adj_relationtototal,
+    cast(inv_adj.inv_adj_total AS money), -- this are adjustments on invoice level
+    cast(inv_adj.inv_adj_total + invl.total AS money) AS invl_total, -- this are costs by invoice_line including adjustments on invoice level
+    inst.instance_format_name,
+    CASE WHEN instform.no_instance_formats = 0 THEN
+             NULL
+         ELSE
+             cast((invl.total + coalesce(inv_adj.inv_adj_total, 0)) / instform.no_instance_formats AS money)
+        END AS "total_by_format", -- this are costs by invoice_line prorated to the number of given formats including adjustments on invoice level
+    inst.instance_subject,
+    CASE WHEN instsub.no_instance_subjects = 0 THEN
+             NULL
+         ELSE
+             cast((invl.total + coalesce(inv_adj.inv_adj_total, 0)) / instsub.no_instance_subjects AS money)
+        END AS "total_by_subject" -- this are costs by invoice_line prorated to the number of given subjects including adjustments on invoice level
+FROM
+    invoice_lines AS invl
+        LEFT JOIN invoice_detail AS inv ON invl.invoice_id = inv.invoice_id
+        LEFT JOIN folio_reporting.invoice_adjustments_ext AS inv_adj ON invl.invoice_id = inv_adj.invoice_id
+        LEFT JOIN po_lines AS pol ON invl.po_line_id = pol.id
+        LEFT JOIN instance_detail AS inst ON pol.instance_id = inst.instance_id
+        LEFT JOIN folio_reporting.po_lines_phys_mat_type AS pol_phys_type ON pol.instance_id = pol_phys_type.pol_id
+        LEFT JOIN folio_reporting.po_lines_er_mat_type AS pol_er_type ON pol.instance_id = pol_er_type.pol_id
+        LEFT JOIN folio_reporting.invoice_lines_adjustments AS invl_adjustments ON invl.id = invl_adjustments.invoice_line_id
+        LEFT JOIN instance_subs AS instsub ON instsub.instance_id = inst.instance_id
+        LEFT JOIN instance_formats AS instform ON instform.instance_id = inst.instance_id
+WHERE
```

Comment:
I got an error here when I ran the query in DBeaver. I think the problem is further down with one of the "AND"s where there's an extra one.